folding/src/standard_config.rs

```diff
@@ -0,0 +1,471 @@
+//! This module offers an standard implementation of [FoldingConfig] supporting
+//! many use cases
+use crate::{
+    expressions::FoldingColumnTrait, instance_witness::Witness, FoldingConfig, FoldingEnv,
+    Instance, Side,
+};
+use kimchi::{circuits::gate::CurrOrNext, curve::KimchiCurve};
+use memoization::ColumnMemoizer;
+use poly_commitment::{commitment::CommitmentCurve, srs};
+use std::{fmt::Debug, hash::Hash, marker::PhantomData, ops::Index};
+
+#[derive(Clone, Default)]
+/// Default type for when you don't need structure
+pub struct EmptyStructure<G: KimchiCurve>(PhantomData<G::ScalarField>);
+
+impl<G: KimchiCurve, Col> Index<Col> for EmptyStructure<G> {
+    type Output = Vec<G::ScalarField>;
+
+    fn index(&self, _index: Col) -> &Self::Output {
+        panic!("shouldn't reach this point, as this type only works with witness-only constraint systems");
```

Comment:
By that, you mean that there is no some kind of selectors?

Reply:
not in any case that makes use of this particular type, as it is only for cases where you don't need structure. I you have selectors you should use your own structure and implement `Index` accordingly.